src/templates/filtersDrawer.jsx

```diff
@@ -0,0 +1,674 @@
+                                            minHeight: '32px',
+                                            width: '32px',
+                                        }}
+                                    >
+                                        <Icon
+                                            compact
+                                            type="check"
+                                        />
+                                    </Button>
+                                    )}
+                                </div>
+                            </div>
+                        )}
+                    />
+
+                    <Drawer.Content>
+                        {_.map(rows, (row) => {
+                            rowKeyNum += 1;
+
+                            return (
+                                <div
+                                    className="page_filters_drawer--row"
+                                    key={`page--filters-drawer-row-${rowKeyNum}`}
+                                >
+                                    {row.header && (
+                                        <Header
+                                            className="page_filters_drawer--header"
+                                            weight="bold"
+                                        >
+                                            {row.header}
+                                        </Header>
+                                    )}
+
+                                    <div
+                                        className="page_filters_drawer--items"
+                                    >
+                                        {/* eslint-disable max-len */}
+                                        {_.isArray(row.items) && _.map(row.items, (item) => {
+                                            const {
+                                                checkbox,
+                                                dropdown,
+                                                jsx,
+                                                multiSelect,
+                                                nestedToggles,
+                                                toggle,
+                                            } = item;
+                                            const itemClassName = ClassNames('page_filters_drawer--item', {
+                                                'page_filters_drawer--item-checkbox': checkbox,
+                                                'page_filters_drawer--item-dropdown': dropdown,
+                                                'page_filters_drawer--item-jsx': jsx,
+                                                'page_filters_drawer--item-multi_select': multiSelect,
+                                                'page_filters_drawer--item-nested_toggles': nestedToggles,
+                                                'page_filters_drawer--item-toggle': toggle,
```

Comment:
Do we need to worry about `page` versus `drawer` here?

Reply:
Fixed.